websockify/websocketproxy.py

```diff
@@ -212,7 +212,10 @@ def do_proxy(self, target):
 
             if self.request in outs:
                 # Send queued target data to the client
-                c_pend = self.send_frames(cqueue)
+                # As do-until loops are not available in python, settings c_pend to true
+                c_pend = True
+                while(c_pend):
+                    c_pend = self.send_frames(cqueue)
```

Comment:
I'm afraid this makes things less responsive, as it will now block here until it can send everything to the browser. That means it will pause sending things in the other direction.